src/components/DownloadInvoiceForm.js

```diff
@@ -9,16 +10,31 @@ export default class DownloadInvoiceForm extends Component {
     this.state = {
       start: moment().startOf('day'),
       end: moment().startOf('day'),
-      client: props.clients.first()
+      client: props.clients.first(),
+      address: '',
+      notes: '',
+      userRates: this.userRatesFromProps(props),
```

Comment:
In the backend, we're just calling it `users` :P
But I feel that it's important to call out the fact that the `rate` key is added, which is not normally the case.